spec/dd-agent_spec.rb

```diff
@@ -295,6 +295,30 @@ def set_env_var(name, value)
           .with_content(/^tags: datacenter:us-foo$/)
       end
     end
+
+    context 'does accept other_api_keys' do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(
+          platform: 'ubuntu',
+          version: '12.04'
+        ) do |node|
+          node.set['datadog'] = {
+            'api_key' => 'somethingnotnil',
+            'other_dd_urls' => ['app.example.com', 'app.datadoghq.com'],
```

Comment:
nitpick since it's not really what's tested here: it'd be better to use the full urls with the protocol, i.e. `https://app.example.com`